src/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include "postprocess.cpp"
+
+AnchorMat create_sample_input() {
+  AnchorMat anchor_vec;
```

Comment:
Please add information what is each anchor vector and what is the collection of anchor vestor supposed to represent

Reply:
This was actually a sample script where I was writing test code. But I have added the Anchor information